reorder self-hosted docs (#716)
run-ai/docs: docs/admin/runai-setup/self-hosted/k8s/backend.md

# Install the Run:ai Control Plane 

## Prerequisites 

> ### Domain certificate

Reviewer: @ozRunAI needs a line between headings

```suggestion

### Domain certificate
```

## (Optional) Additional Configurations

> #### Helm values

Reviewer: @ozRunAI is this supposed to be a H4? Maybe you meant an H3?
```suggestion
### Helm values
```

| `grafana.adminUser`  | Grafana username  |   Override the Run:ai default user name for accessing Grafana |
| `grafana.adminPassword`  | Grafana password  |   Override the Run:ai default password for accessing Grafana |
| `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
| `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
|<div style="width:200px"></div>| | |

Use the `--set` syntax in the helm command above.  

#### Custom docker registry credentials 

> To access the organization's docker registry it is required to set the registry's credentials (imagePullSecret)

Reviewer: @ozRunAI missing period at the end of the sentence.
```suggestion
To access the organization's docker registry it is required to set the registry's credentials (imagePullSecret).
```

> Create the secret named `runai-reg-creds` based on your existing credentials as explained [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials){target=_blank}.

Reviewer: @ozRunAI we try not to use the word [here] with a link.
```suggestion
Create the secret named `runai-reg-creds` based on your existing credentials. For more information, see [Create a Secret based on existing credentials](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials){target=_blank}.
```

| `grafana.adminUser`  | Grafana username  |   Override the Run:ai default user name for accessing Grafana |
| `grafana.adminPassword`  | Grafana password  |   Override the Run:ai default password for accessing Grafana |
| `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
| `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
|<div style="width:200px"></div>| | |

Use the `--set` syntax in the helm command above.  

## Next Steps

> ### Connect to Run:ai User Interface

Reviewer: ```suggestion

### Connect to Run:ai User Interface
```
